### PR TITLE
Add docs for accepted and relevant filters

### PR DESCRIPTION
**Pull request by JeffBain**

### CODE REVIEW

File: source/includes/tasks.md [hunk 1]
```diff
@@ -167,12 +167,52 @@ related   | Includes a list of related tasks
...
+    }]
+}
+```
 
+You can filter the tasks returned by the task list endpoint by their relevance
+and whether or not they have been accepted into a project by a project lead.
+If no filters are passed, we default to returning accepted tasks to match with
+the list shown in the web application.
 
-
```
Comment (geoffwhittington) [inline source/includes/tasks.md]: When i preview this, the table doesn't show up right. Is this correct?

Comment (JeffBain) [inline source/includes/tasks.md]: Did you manage to actually view it? I was working a little blind here due to being unable to regenerate the docs thanks to some of the libraries not installing. It's why this has sort of been in limbo.

Comment (geoffwhittington) [inline source/includes/tasks.md]: I am simply using the preview feature on github to view the markdown... maybe that is not the correct way?

Comment (hhaidar) [inline source/includes/tasks.md]: screen shot 2016-03-10 at 12 19 42 am

Ya you should definitely get slate running locally. If you need help with rvm/gems, bug me tomorrow.

File: source/includes/tasks.md [hunk 2]
```diff
@@ -165,14 +165,54 @@ how_tos   | Includes a list of applicable how-tos
...
+        "library_task_updated": "2015-06-16T19:36:57.836874Z",
+        "verification_status": null,
+        "status": "TS2",
+        "note_count": 0,
+        "artifact_proxy": null,
+    }]
+}
+```
 
+You can filter the tasks returned by the task list endpoint by their relevance
```
Comment (hhaidar) [inline source/includes/tasks.md]: Maybe something like:

> Tasks can be filtered by their relevance and whether or not they have been accepted into a project by a project lead.

**Comment by funkaoshi** [comment]
@JeffBain Are these changes already live, or in the next release?

**Comment by geoffwhittington** [comment]
These are live in 3.6

**Comment by geoffwhittington** [comment]
Good to merge?

**Comment by funkaoshi** [comment]
@JeffBain can you fix the suggestion Houssam made and then merge this.

**Comment by geoffwhittington** [comment]
@JeffBain can you fix the conflict and merge?

**Comment by funkaoshi** [comment]
This is in 3.7, maybe 3.6.

**Comment by geoffwhittington** [comment]
This was in 3.6

**Pull request merged**